hw/ip/pwrmgr/data/pwrmgr.hjson

```diff
@@ -70,6 +77,30 @@
   regwidth: "32",
   registers: [
 
+    { name: "CTRL_CFG_REGWEN",
+      swaccess: "ro",
```

Comment:
@tjaychen sorry for the late comment
Is this reg "RO" for SW? It's a bit weird for write enable register.
Is the description "Once set to 0, it can no longer be configured to 1" true for field "EN"? HW will only set it to 0.

@cindychip for visibility